doc/install/install-kubernetes.md

```diff
@@ -0,0 +1,135 @@
+# Install Kuadrant on a Kubernetes cluster
+
+> [!NOTE]  
+> You must perform these steps on each Kubernetes cluster where you want to use Kuadrant.
+
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, with `kubeadmin` or an account with similar permissions
+
+## Procedure
+
+This guide will show you how to install Kuadrant onto a bare Kubernetes cluster.
+
+Alternatively, if you are looking instead for a way to set up Kuadrant locally to evaluate or develop, consider running the kind & Kubernetes [quickstart script](https://docs.kuadrant.io/latest/getting-started-single-cluster/).
+
+### Install Gateway API
+
+```bash
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+```
+
+### Install [OLM](https://olm.operatorframework.io/)
+
+> [!NOTE]  
+> Currently, we recommend installing our operator via OLM. We plan to support Helm soon.
+
+```bash
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.23.1/install.sh | bash -s v0.23.1
+```
+
+### Install Istio as a Gateway API provider
```

Comment:
we might need to add an extra step for the cert-manager if https://github.com/Kuadrant/kuadrant-operator/pull/727 gets merged

Reply:
👍  will wait for yours to land and some docs to point towards cert-manager install

Reply:
Resolving as https://github.com/Kuadrant/kuadrant-operator/pull/710/files#diff-bbf82f7e8d643b7752faaf388f879d2838aa6db4a985c4664663f6d3a451443bR10 has been added